pkg/cases/handler.go

```diff
@@ -8,12 +8,39 @@ import (
 type CaseHandler struct {
 }
 
+// @Summary Get the lastest case ID
+// @Description
+// @Tags Case
+// @Produce json
+// @Param patientID path uint true "patient ID"
+// @Success 200 {integer} api.ReturnedData{data=uint}
+// @Router /{patientid}/case [GET]
```

Comment:
`/patient/{patientID}/case` may be better. `/api/10/case` is somewhat confusing, what does that 10 mean?

Reply:
And a case ID is quite useless, what about directly retrieve the case?